dashboard/config/locales/en.yml

```diff
@@ -1404,9 +1404,25 @@ en:
     founder: "Founder, Code.org"
     your_friends: "Your Friends at Code.org"
   lti:
-    create_integration_success: "Succesfully created your LTI integration"
+    create_integration_success: "Successfully created your LTI integration"
     error:
       integration_exists: "Lti Integration already exists for this Client ID"
-      missing_params: "Missing required param(s): client_id, lms, email"
+      missing_params: "Missing required param(s): Client ID, LMS, Email"
       unsupported_lms_type: "Unsupported LMS platform type"
       wrong_resource_type: "Only LtiResourceLink is supported right now"
+    integration_client_id: "LMS Client ID"
+    integration_email: "Your email"
+    integration_navigate_home: "Back to Code.org"
+    integration_navigate_registration: "Back to Registration"
+    integration_lms_selector: "What LMS are you using?"
+    integration_portal_exists_body: "It seems like your LMS is already in our system. You should be able to continue installing and using our LTI integration."
+    integration_portal_exists_header: "Looks like you're are already registered!"
+    integration_portal_guide_instructions: "Full guides for installing and our LTI integration <a href=''>can be found here.</a> Running into issues? Feel free to <a href=''>contact us</a> for additional support."
+    integration_portal_header: "Register Your LMS with Code.org"
+    integration_portal_instructions: "Please use this form to register your Learning Management System (LMS) with Code.org. This will allow you to install our LTI integration on your LMS."
+    integration_portal_instructions_2: "Full guides for obtaining your LMS Client ID, using LMS/LTI integrations, and our list of supported Learning Management Systems <a href='https://code.org/contact'>can be found here.</a>"
+    integration_portal_instructions_3: "Not seeing your LMS in our list, or running into issues with your installation? Check out our <a href=''>guides</a>, or <a href='https://code.org/contact'>contact us</a> for additional help"
+    integration_portal_privacy_policy: "Your privacay is of utmost importance to us. Your personal information will not be used for any marketing purposes. Please see our <a href='https://code.org/contact'>Privacy Policy here</a> for more information."
+    integration_portal_success_body: "Great news! Your LMS has been registered with Code.org, and you can continue installing and using your LMS integration. You should receive a confirmation email shortly."
+    integration_portal_success_header: "Thank you for registering your LMS!"
```

Comment:
I'm not sure if this would make any difference, but you could optionally add a hierarchy.
```
integration_portal
    exists_body: "..."
    exists_header: "..."
```

Reply:
Oh good call. It would at least shorten the strings, which would be a plus.